packages/dds/tree/src/tableSchema.ts

```diff
@@ -244,12 +244,11 @@ export namespace System_TableSchema {
 	> = OptionsWithSchemaFactory<TSchemaFactory> & OptionsWithCellSchema<TCell>;
 
 	/**
-	 * Factory for creating new table row schema.
-	 *
+	 * Factory for creating row schema.
 	 * @sealed @internal
 	 */
 	// eslint-disable-next-line @typescript-eslint/explicit-function-return-type -- Return type is too complex to be reasonable to specify
-	export function createRowInternal<
+	export function createRowSchema<
```

Comment:
Renamed internal function variants to
a. Not be unnecessarily postfixed with "Internal"
b. Be more explicit that they are creating _schema_.